src/components/MMSearch/MMSearch.component.tsx

```diff
@@ -126,29 +126,57 @@ export const MMSearch = (props: MMSearchProps) => {
             setActive(0);
             return;
         }
-        if (event.key === 'ArrowUp') {
+        if (event.key === Constants.KeyboardEvent.ARROW_UP) {
             event.preventDefault();
             if (active === 0) {
                 return;
             }
             setActive((prev) => prev - 1);
-            if (typeof listRef.current?.scrollBy === 'function') {
-                listRef.current.scrollBy(0, -Constants.ITEM_HEIGHT);
-            }
+
+            listRef.current.scrollBy(0, -Constants.ITEM_HEIGHT);
```

Comment:
What if `listRef.current` or `listRef.current.scrollBy` is undefined?

Reply:
Added